advanced/Exercise_7.md

```diff
@@ -1,154 +1,97 @@
+In this exercise, you'll practice implementing a pre-commit hook in the `conference_planning` repository to prevent committing files with trailing whitespace. You'll also learn how to set up multiple pre-commit hooks for different tasks.
 
 This exercise uses the same git repository that was created in Exercise 3. If you have not already done so, you can create it by following the instructions in the ["Initialize the git repository" section of Exercise 3](./Exercise_3.md#initialize).
 
-* [Fork a repository on Github and add it as a submodule](#submodule)
+* [Navigate to the Repository](#navigate)
 
-* [Examine how git deals with changes to both repositories](#examine)
+* [Implement Pre-Commit Hook for Trailing Whitespace](#whitespace)
 
-* [Create new content locally and update the sub-repository](#push)
+* [Test the Hook](#test)
 
-* [Create new content in the sub-repository and update the local repository](#pull)
+* [Add Another Custom Hook](#another)
 
-## Fork a repository on Github and add it as a submodule <a name="submodule"></a>
+* [Conclusion](#conclusion)
 
-Navigate to the folder containing the `conference_planning` folder; in other words you should be in a folder that does not contain a git repository. Make a copy of the conference planning repository to use for adding the submodule.
+## Navigate to the Repository <a name="navigate"></a>
 
-```plaintext
-cp -r conference_planning conference_submodule
-```
-
-I have already created a repository on Github that we will use as a submodule. This repository is called `posters` and will contain the poster schedule for our conference.
-
-Navigate to the repository, found [here](https://github.com/kosterried/posters), and make a fork of it using the Github user interface.
-
-Let's add this fork as a submodule to our conference planning repository. You will need to copy the SSH address of your fork and paste it into the `git submodule add` command below.
+Open a terminal and navigate to the `conference_planning` repository on your local machine.
 
-```plaintext
-cd conference_submodule
-git submodule add YOUR_FORK_ADDRESS
-```
-
-Note that Github requires an SSH key in order to push content to repositories. If you have not already set up an SSH key in your Github account, it is easy to do and you can find instructions [here](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account).
+## Implement Pre-Commit Hook for Trailing Whitespace <a name="whitespace"></a>
 
-Check the status of the repository.
+Create a new file named `pre-commit` in the `.git/hooks` directory. Make it executable.
 
-```plaintext
-git status
+```sh
+touch .git/hooks/pre-commit
+chmod +x .git/hooks/pre-commit
```

Comment:
Should we say something about `touch` and `chmod`? I'm sure there are people who don't know what that is.

Reply:
`touch` has been used in several exercises before. I will add a comment about `chmod +x".